### PR TITLE
Add "deployment and platform options" from dagobah training

### PR DESCRIPTION
**Pull request by bgruening**
ping @nsoranzo

### CODE REVIEW

**Comment by nsoranzo** [comment]
Ping @natefoo, I suppose.

File: topics/admin/tutorials/deployment-platforms-options/slides.html [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+  - All major distros in wide circulation have 2.7, *except* RHEL<sup>[1]</sup> 6
+    - See: Software Collections for [RHEL](https://access.redhat.com/solutions/472793), [CentOS](https://wiki.centos.org/AdditionalResources/Repositories/SCL), [Scientific Linux](http://linux.web.cern.ch/linux/scl/)
+
+Optional (but not really):
+  - PostgreSQL
+    - Covered: Today
+  - uWSGI (will come with Galaxy from 18.01 onwards)
+    - Covered: Tuesday
+  - Reverse proxy server (nginx, Apache)
+    - Covered: Today
```
Comment (nsoranzo) [inline topics/admin/tutorials/deployment-platforms-options/slides.html]: I'd removed the references to Today/Tuesday

File: topics/admin/tutorials/deployment-platforms-options/slides.html [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+- Figure out where Galaxy will be stored
+  - Make sure it will be accessible to any eventual compute
+- Figure out where data will be stored
+  - Make sure it will be accessible to any eventual compute
+
+---
+# Deployment Best Practices
+
+**Use configuration management**
+- Ansible (covered on Wednesday)
```
Comment (nsoranzo) [inline topics/admin/tutorials/deployment-platforms-options/slides.html]: Here too.

File: topics/admin/tutorials/deployment-platforms-options/slides.html (hunk 1 above)
Comment (bgruening) [reply topics/admin/tutorials/deployment-platforms-options/slides.html]: What about including links to the mentioned training?

Comment (nsoranzo) [reply topics/admin/tutorials/deployment-platforms-options/slides.html]: Sure (together with removing the days).

**Pull request merged**